freqtrade/optimize/hyperopt.py

```diff
@@ -661,6 +681,14 @@ def start(self) -> None:
             with Parallel(n_jobs=config_jobs) as parallel:
                 jobs = parallel._effective_n_jobs()
                 logger.info(f'Effective number of parallel workers used: {jobs}')
+
+                # Define progressbar
+                self.progress_bar = tqdm(
+                    total=self.total_epochs, ncols=108, unit=' Epoch',
+                    bar_format='Epoch {n_fmt}/{total_fmt} ({percentage:3.0f}%)|{bar}|'
+                    ' [{elapsed}<{remaining} {rate_fmt}{postfix}]'
+                )
```

Comment:
```suggestion
                self.progress_bar = tqdm(
                    total=self.total_epochs, unit='Epoch',
                    bar_format='Epoch {n_fmt}/{total_fmt} ({percentage:3.0f}%) |{bar}|'
                               ' [{elapsed}<{remaining} {rate_fmt}{postfix}]',
                    ncols=108,
                    position=1,
                    leave=False,
                )
```

Reply:
I'm a bit sceptical about the leave=False
1. We can't do a pytest to see that it prints
2. If we remove the {current}/{total epoch} we do not have any indicator at the end showing how many it was suppose to perform?

Reply:
maybe, need to think of the resulting behavior...

you can remove `leave=False` for a while, I'll commit the further proposals for review...

Reply:
For me it prints correct now with the -v option but I also have some extra code. 
Need to check with this new commit

Reply:
not quite:
run it with `-j 1 -v` (to see a shorter logging message first at each epoch) -- you'll see that the progressbar is not cleared. Same if you add printing some short value in the populate_buy_trend in the hyperopt class used.

Reply:
this is solved in the latest pull using progressbar2